README.md

```diff
@@ -40,7 +40,7 @@ See: https://github.com/betaflight/betaflight/wiki
 
 There's a dedicated Slack chat channel here:
 
-http://www.betaflight.ch/
+https://slack.betaflight.tech/
```

Comment:
Should be https://slack.betaflight.com/ I think.

Reply:
agreed :) that adress wasnt there when i created this pr.
will update it later

Reply:
The configurator uses http://www.betaflight.tk at some places, I suppose it will need to use the new .com too, right?

Reply:
yes. .com should be used everywhere.. (beside the jenkins which will stay .tech)